insecure/corruptlibp2p/pubsub_adapter_config.go

```diff
@@ -85,10 +92,10 @@ func (c *CorruptPubSubAdapterConfig) Build() []corrupt.Option {
 	return c.options
 }
 
-func defaultCorruptPubsubOptions(base *p2p.BasePubSubAdapterConfig, withMessageSigning, withStrictSignatureVerification bool) []corrupt.Option {
```

Comment:
I missed this in my initial review, why were these options removed ?

Reply:
reverted 77c9dde79e6503b00f78c0f7ab2d6ffd2c36990b